server/src/main/java/org/opentosca/toscana/plugins/kubernetes/KubernetesLifecycle.java

```diff
@@ -0,0 +1,36 @@
+package org.opentosca.toscana.plugins.kubernetes;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
+
+public class KubernetesLifecycle extends AbstractLifecycle {
+    /**
+     @param context because the context is always needed this should never be null
+     It probably gets called by the <code>getInstance</code> method of the LifecycleAwarePlugin
+     */
+    public KubernetesLifecycle(TransformationContext context) throws IOException {
+        super(context);
+    }
+
+    @Override
+    public boolean checkModel() {
+        return false;
```

Comment:
Not implemented methods should always `throw new UnsupportedOperationException()` instead of return `null` or `false`.
Pros are: Not implemented functions sometimes are forgotten; later you excounter an NullPointerexception somewhere, or you wonder why a particular thing doesn't work -- best practice is to always throw this default exception.